src/pages/sighting/SightingCore.jsx

```diff
@@ -146,16 +170,15 @@ export default function SightingCore({
         onClearError={
           pending ? deleteAsgOnClearError : deleteSightingOnClearError
         }
-        messageId="CONFIRM_DELETE_SIGHTING_DESCRIPTION"
+        messageId={messageForConfirmDelete}
       />
       <SightingEntityHeader
         activeTab={activeTab}
         data={data}
         loading={loading}
         pending={pending}
         preparing={isPreparationInProgress}
-        guid={id}
-        // setHistoryOpen={setHistoryOpen}
+        guid={id} // setHistoryOpen={setHistoryOpen}
```

Comment:
For me:
1. prettier doesn't even run because there are existing lint violations in this file.
2. When I disable the lint violations, prettier still does not move this comment.

If possible, this comment should be left on the new line.

Reply:
... I have no idea what happened here, but husky/prettier/etc. all seem fine _either way_ with me. Hmm. Setting to its own line...